res/database/Default/database_news.xml

```diff
@@ -93,11 +93,11 @@
 		<news id="75e966f2-929d-479d-b24a-9ef2fcbf529c" thread_id="0328d075-c155-43c9-b0c1-e130eb972f38" type="2" creator="">
 			<title>
 				<de>Start der Shuttles in der Ukraine</de>
-				<en>Lift-off of shuttles in Ukraine</en>
+				<en>Launch of the shuttles in Ukraine</en>
 			</title>
 			<description>
-				<de>Die US Shuttles zur Terrorismusbekämpfung im All werden vom Ukrainischen Hafen „Bikonur“ starten. „Das ist ein gutes Signal für die Sicherheit der Welt“, so Baush.</de>
-				<en>The US shuttles to fight terrorism in space will start from the Ukrainian port "Bikonur". "This is a good signal for the security of the world", said Baush.</en>
+				<de>Die US-Shuttles zur Terrorismusbekämpfung im All werden vom Ukrainischen Hafen „Bikonur“ starten. „Das ist ein gutes Signal für die Sicherheit der Welt“, so Baush.</de>
+				<en>The US shuttles to fight terrorism in space will launch from the Ukrainian port of "Bikonur." "This is a good signal for the security of the world," Baush said.</en>
```

Comment:
think the dot should stay outside of the quotation marks. `"Bikonur."` -> `"Bikonur".`

Reply:
No, I'll be changing that, that was not intended, the translation tool did it that way.
I'm almost certain it's *not* correct to put the dot inside when you just put a single term in quotes.
When you have a proper citation or direct speech, sure, then that's a different story, but even then, almost no-one follows that rule in the internet age.